tools/repl/swift/REPLTime/REPLTime.h

```diff
@@ -0,0 +1,38 @@
+// REPLTime.h
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+// -----------------------------------------------------------------------------
+
+#ifndef REPL_TIME_H
+#define REPL_TIME_H
+
+typedef unsigned long long uint64_t;
```

Comment:
It *might* work just because you literally don't import anything else, but it's pretty brittle.